airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -283,7 +283,7 @@ def _get_ti_pod_labels(context: Optional[dict] = None, include_try_number: bool
             return {}
 
         ti = context['ti']
-        run_id = getattr(ti, 'run_id') or context['run_id']
+        run_id = getattr(ti, 'run_id', None) or context['run_id']
```

Comment:
I think this one’s mentioned in another PR; it should probably just be `context["run_id"]` since the key is available since basically forever.

Reply:
You are quite ritght. Actually I figured that even for 2 months it is worth to add an extra protection and pre-commit to check it and - as it turned out  there was one more case like that in elasticsearch which I also fixed.